0065-drop-dunder-utils.md

```diff
@@ -0,0 +1,84 @@
+- Feature Name: deprecate-dunder-utils
+- Start Date: 20022-06-23
+- SEP Status: Draft
+- SEP PR: (leave this empty)
+- Salt Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Deprecate and remove all usage of the ``__utils__`` Salt loader.
+
+# Motivation
+[motivation]: #motivation
+
+## Why are we doing this? What use cases does it support? What is the expected outcome?
+
+The Salt loader is a **controlled** hack around Python's import system which allows
+loading(or prevent loading) of python modules which contribute to the Salt ecosystem:
+
+* Execution modules
+* State execution modules
+* Runners
+* etc ...
+
+The **controlled** part of the loader is extremely important and the Salt team
+goes to great lengths to maintain that controlled environment bug free and working
+as supposed.
+
+The ``__utils__`` loader was added to facilitate the utility functions usage.
+For example, it helps by reducing the ammount of imports at the top of the module.
+In the case where a utility module is targetted to a specific platform(for example,
+Windows), we can prevent that utility module from loading on the wrong platform.
+
+While all of this suggests it was a great idea, Salt lost the **controlled** part
```

Comment:
Can you elaborate on what being "**controlled**" actually means? What is controlled? What are the restrictions that must be met?

Reply:
The issues linked explain the uncontrolled(and uncontrollable) side effects.